foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -339,10 +339,19 @@ AEPIdentityItem *item = [[AEPIdentityItem alloc] initWithId:@"user@example.com"
 
 ## resetIdentities
 
-Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\) . Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\). Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+
+This is a destructive action. Once an ECID is removed it cannot be reused. The new ECID generated by this API can impact metrics like unique visitors and cause duplication of user profiles and device profiles. To remove existing custom identifiers, use [`removeIdentity`](#removeidentity).
```

Comment:
Rewording suggestion: This is a destructive action since once an ECID is removed it cannot be reused.